layouts/shortcodes/docs/prog-lang-home-content.html

```diff
@@ -0,0 +1,25 @@
+{{ $lang := lower ($.Page.Params.language | default $.Page.Params.title) }}
```

Comment:
Nice solution!

Shouldn't we be able to give this file a `.md` suffix given that it is rendering markdown?

(And if so, the same would be true of any other shortcodes with a `.html` suffix that is actually being used to render markdown.)

Reply:
This won't break if you change the file type, however I'm not sure about Hugo's opinions on best practices here. I haven't been able to find much on the topic. I gave this an `.html` type though, because it's largely still concerned with the HTML structure of the lists, and because comparable shortcodes [using the `%`] in Docsy are using the `.html` type. [e.g. `blocks/feature.html`]

Reply:
> This won't break if you change the file type, ...

Actually, since you're going to be making changes to this PR, could you change the suffix on this file to `.md`?